src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -49,6 +49,12 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.fullFunctionalTest = true
   }
 
+  void enableFullFunctionalTestNonServiceApp(String nonServiceAppTestUrl, int timeout = 30) {
```

Comment:
I recommend just overloading `enableFullFunctionalTest` method and pass testURL as optional. It will be useful even in  someone wants to override in normal pipeline.